frontend/src/components/Input/Input.tsx

```diff
@@ -0,0 +1,56 @@
+import {
+  forwardRef,
+  Icon,
+  Input as ChakraInput,
+  InputGroup,
+  InputProps as ChakraInputProps,
+  InputRightElement,
+  useMultiStyleConfig,
+} from '@chakra-ui/react'
+import { omit } from '@chakra-ui/utils'
+
+import { BxsCheckCircle } from '~assets/icons/BxsCheckCircle'
+
+export interface InputProps extends ChakraInputProps {
+  /**
+   * Whether the input is in a prefilled state.
+   */
+  isPrefilled?: boolean
+  /**
+   * Whether the input is in a success state.
+   */
+  isSuccess?: boolean
+}
+
+export const Input = forwardRef<InputProps, 'input'>((props, ref) => {
+  const inputStyles = useMultiStyleConfig('Input', props)
+
+  // Omit extra props so they will not be passed into the DOM and trigger
+  // React warnings.
+  const inputProps = omit(props, ['isSuccess', 'isPrefilled'])
+
+  // Return normal input component if not success state.
+  if (!props.isSuccess) {
+    return <ChakraInput ref={ref} {...inputProps} sx={inputStyles.field} />
+  }
```

Comment:
sorry why are we doing this?

Reply:
As per the comments in the non-success case, `InputGroup` is required for `InputRightElement` to retrieve the correct style props and will crash if not included. 

However, since we just display the normal input on success, we do not need that special requirement and can just return the input as per normal